### DISCUSSION TITLE
Jenkins documentation jobs still have -v3 suffix

### DISCUSSION THREAD

**Discussion by wborn** [openhab/openhab-docs#2058] [Team Posts]
Hi @openhab/documentation-maintainers!

Do you think the Jenkins documentation jobs should be updated now that we are working on v4?

These jobs have a `-v3` suffix:

* gather-external-docs-v3
* merge-docs-v3
* redeploy-stable-docs-v3
* redeploy-website-v3

See: https://ci.openhab.org/view/Documentation/

Maybe the suffix should be removed?

**Comment by Confectrician** [comment]
I would remove the suffix completely.
We are only using "stable" for one job and all our other jobs are alway based on the "latest" docs.

So it always clear without any suffix.
I will take care of it. :)
